modules/runtime/src/service/systemd/xp.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Enonic XP
+Documentation=https://developer.enonic.com/docs
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+PrivateTmp=true
+Environment=XP_INSTALL=/opt/enonic/xp
+Environment=XP_JAVA_HOME=${XP_INSTALL}/jdk
+
+WorkingDirectory=/opt/enonic/xp
+
+User=xp
+Group=xp
+
+ExecStart=/opt/enonic/xp/bin/server.sh
+
+StandardOutput=journal
+StandardError=inherit
+
+LimitNOFILE=65535
+
+LimitNPROC=4096
+
+LimitAS=infinity
+
+LimitFSIZE=infinity
+
+KillSignal=SIGTERM
+
+KillMode=process
+
+SendSIGKILL=no
+
+SuccessExitStatus=143
```

Comment:
does this apply to sh script? we don't directly call java, but does script propagate same java exit code (143 means java exited normally after SIGTERM)

Reply:
@anatol-sialitski can you verify this, please

Reply:
Doing a test on my machine:

```console
$XP_HOME=$HOME/.enonic/sandboxes/jwt/home ./bin/server.sh

....

CTRL+C

$echo $?
130
```

Reply:
Killing with sigterm results in 143.